TeamPage.html

```diff
@@ -566,6 +566,55 @@ <h2>Meet Our Team</h2><br>
               </div>
             </div>
           </div>
+        </div>
+        <div class="col-sm-3 col-xs" style="min-width: 210px;">
+          <div class="id_presentation">
+            <img class="img_staff" src="https://avatars.githubusercontent.com/u/153776576?v=4"
+              style="width: 200px; height: 200px; object-fit: cover;" alt="Stacey's Picture" />
+            <div class="text_pres">Stacey Lai<br>
+              <div class="subtext_pres">Student Programmer</div>
+            </div>
+            <div class="container" style="text-align: center;">
+              <button type="button" style="margin-top: 10px;" class="btn btn-danger btn-sm" data-toggle="modal"
+                data-target="#myModalStaceyL" onclick="openBSModal('myModalStaceyL', this)">Stacey's
+                Profile</button>
+              <div class="modal fade" tabindex="-1" id="myModalStaceyL" role="dialog" datim-modal>
+                <div class="modal-dialog">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <div class="modal-title">
+                        <p class="text-justify"><b>Stacey Lai</b> - Student Programmer<br>
+                          Work Location: Remote<br>
+                          Email: stacey.lai@unlv.edu</p>
+                      </div>
+                      <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
+                    </div>
+                    <div class="modal-body">
+                      <p class="text-justify">Stacey Lai is in her 3rd year working towards a B.S. computer science degree, studying at 

```

Comment:
Minor suggestion here if you see it appropriate:
> ... towards a B.S. in Computer Science degree, ...

Reply:
@Kelvyn-Meyers Hi! It's fixed now!